Functions.Templates/Templates/CosmosDBTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "defaultFunctionName": "CosmosTriggerCSharp",
+  "description": "$CosmosDBTriggerCSharp_description",
+  "name": "CosmosDBTrigger",
+  "language": "C#",
+  "filters": [ "cosmosDb" ],
+  "category": [
+    "$temp_category_core",
+    "$temp_category_dataProcessing"
```

Comment:
can you remove temp_category_core & temp_category_dataProcessing and add temp_category_experimental instead